zerver/webhooks/intercom/view.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+from typing import Any, Dict, Iterable, Text
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.actions import check_send_stream_message
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.models import UserProfile
+
+@api_key_only_webhook_view('Intercom')
+@has_request_variables
+def api_intercom_webhook(request: HttpRequest, user_profile: UserProfile,
+                         payload: Dict[str, Any] = REQ(argument_type='body'),
+                         stream: Text = REQ(default='test')) -> HttpResponse:
+    topic = str(payload['topic'])
+    topic = topic.replace('.', ' ').capitalize()
+
+    created_at = datetime.datetime.fromtimestamp(int(payload['created_at'])).strftime('%H:%M:%S %Y-%m-%d')
+    body = '*{created_at}* **{topic}**: \n'.format(topic=topic, created_at=created_at)
+
+    if topic == 'User tag created':
+        body += user_created(payload['data']['item']['user'])
+
+    check_send_stream_message(user_profile, request.client, stream, topic, body)
+    return json_success()
+
+def user_created(data: Dict[str, Any]) -> str:
+    body = ''
+
+    body += ' - User Name: {}\n'.format(data['name'])
```

Comment:
There are several pretty ways of writing a multiline string. I usually enclose all lines with (). That way, you can avoid the repetitive body +=

Reply:
Done!